scripts/update.sh

```diff
@@ -16,6 +16,11 @@ case $key in
     echo "$USAGE" >&2
     exit 1
     ;;
+    --stack-name)
+    STACKNAME="$2"
```

Comment:
question: which is the stack name? Since PCM has multiple nested stacks are we talking about the "main" one? I guess so..